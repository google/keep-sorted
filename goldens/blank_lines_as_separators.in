// keep-sorted-test start by_regex=(k[A-Za-z0-9_]+)(?:Name|Description),(Name|Description) prefix_order=Name skip_lines=2 block=yes blank_lines_as_separators=yes

kAccelerateName="Accelerate"
kAccelerateDescription="When pressed, moves faster."

kTurnName="Turn left or right"
kTurnDescription="When used, changes direction."

kDecelerateDescription="When pressed, brakes."
kDecelerateName="Decelerate"

// keep-sorted-test end
