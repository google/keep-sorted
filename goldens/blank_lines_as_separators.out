// keep-sorted-test start by_regex=(k[A-Za-z0-9_]+)(?:Name|Description),(Name|Description) prefix_order=Name skip_lines=2 block=yes blank_lines_as_separators=yes

kAccelerateName="Accelerate"
kAccelerateDescription="When pressed, moves faster."

kDecelerateName="Decelerate"
kDecelerateDescription="When pressed, brakes."

kTurnName="Turn left or right"
kTurnDescription="When used, changes direction."

// keep-sorted-test end
